noseiquela_orm/client.py

```diff
@@ -1,26 +1,29 @@
-from typing import Callable, Dict, Optional, Union
+from typing import TYPE_CHECKING
 from functools import partial
 
 from google.cloud import datastore
 from google.cloud.datastore.client import _CLIENT_INFO
 from google.cloud.datastore.entity import Entity as GoogleEntity
-from google.auth.credentials import Credentials as GoogleCredentials
-from google.api_core.gapic_v1.client_info import ClientInfo as GoogleClientIngo
-from google.api_core.client_options import ClientOptions as GoogleClientOptions
-from google.api_core.retry import Retry as GoogleRetry
-from requests import Session
 
+if TYPE_CHECKING:
+    from typing import Callable, Dict, Optional, Union
+    from google.auth.credentials import Credentials as GoogleCredentials
+    from google.api_core.gapic_v1.client_info import ClientInfo as GoogleClientIngo
+    from google.api_core.client_options import ClientOptions as GoogleClientOptions
+    from google.api_core.retry import Retry as GoogleRetry
+    from requests import Session as HttpSession
 
-class DataStoreClient:
+
+class DatastoreClient:
     def __init__(self,
-        project:Optional[str]=None,
-        namespace:Optional[str]=None,
-        credentials:Optional[GoogleCredentials]=None,
-        client_info:Optional[GoogleClientIngo]=None,
-        client_options:Optional[GoogleClientOptions]=None,
-        _http:Optional[Session]=None,
-        _use_grpc:Optional[bool]=None
-    ) -> None:
+        project: 'Optional[str]'=None,
+        namespace: 'Optional[str]'=None,
+        credentials: 'Optional[GoogleCredentials]'=None,
+        client_info: 'Optional[GoogleClientIngo]'=None,
+        client_options: 'Optional[GoogleClientOptions]'=None,
+        _http: 'Optional[HttpSession]'=None,
+        _use_grpc: 'Optional[bool]'=None
+    ) -> 'None':
```

Comment:
I put all the type hints inside (simple) quotes so that the dependencies are only really imported when `TYPE_CHECKING` is `True`.